### PR TITLE
Added SDL2 bindings from veandco.

### PR DESCRIPTION
**Pull request by andreas-t-jonsson**

### CODE REVIEW

**Comment by shurcooL** [comment]
LGTM.

File: README.md [hunk 1]
```diff
@@ -340,6 +340,7 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 * [glop](https://github.com/runningwild/glop) - Glop (Game Library Of Power) is a fairly simple cross-platform game library.
 * [go-astar](https://github.com/beefsack/go-astar) - Go implementation of the A* path finding algorithm
 * [go-collada](https://github.com/GlenKelley/go-collada) - Go package for working with the Collada file format.
+* [go-sdl2](https://github.com/veandco/go-sdl2) - Go bindings for the [Simple DirectMedia Layer](http://www.libsdl.org/).
```
Comment (shurcooL) [inline README.md]: Actually, can you please update the https://www.libsdl.org/ link to be `https://` rather than `http://`? It's 2015.

**Comment by andreas-t-jonsson** [comment]
Done

**Comment by shurcooL** [comment]
LGTM.

**Pull request merged**